pkg/kubelet/cm/devicemanager/device_plugin_stub.go

```diff
@@ -96,10 +105,20 @@ func (m *Stub) Start() error {
 	return nil
 }
 
-// Stop stops the gRPC server
+// Stop stops the gRPC server. Can be called without a prior Start
+// and more than once. Not safe to be called concurrently by different
```

Comment:
If Start is not called, do we still need to call Stop? If we make sure Stop is only called if Start has been called, can we remove stopIsOpen?